srcv2/components/PlanFundingOnboarding.js

```diff
@@ -0,0 +1,169 @@
+            console.log('err')
+            console.log(err)
+        }
+        return true
+    }
+
+    const renderCurrencies = () => {
+        const optionClick = (currency) => {
+            setOpenCurrencyOpts(false)
+            setCurrency(currency)
+        }
+        return CURRENCIES.map(currency => {
+            return (
+                <button 
+                    type='button'
+                    onClick={() => optionClick(currency)}
+                    className='text-gray-700 block px-4 py-2 text-sm text-ellipsis overflow-hidden'
+                    role='menuitem'
+                    tabIndex='-1'
+                    id='menu-item-0'
+                    key={currency.name}
+                >
+                    {currency.name}
+                </button>    
+            )
+        })
+    }
+
+    const renderTimeFrameOpts = () => {
+        const optionClick = (timeframe) => {
+            setOpenTimeFrameOpts(false)
+            setTimeFrameAmount(timeframe)
+        }
+        return FUNDING_PLAN_TIMEFRAME_AMOUNTS.map(timeframe => {
+            return (
+                <button 
+                    type='button'
+                    onClick={() => optionClick(timeframe)}
+                    className='text-gray-700 block px-4 py-2 text-sm text-ellipsis overflow-hidden'
+                    role='menuitem'
+                    tabIndex='-1'
+                    id='menu-item-0'
+                    key={timeframe}
+                >
+                    {timeframe}
+                </button>    
+            )
+        })
+    }
+    
+    return (
+        <div>
+            <Section>
+                <div className='grid grid-cols-1 gap-2'>
+                    <p className='text-3xl text-center font-bold'>
+                        Plan funding
+                    </p>
+                    <p>
+                        Lorem Ipsum is simply dummy text of the printing and typesetting
+                    </p>
+                </div>
+            </Section>
+            <Section>
+                <div className='grid grid-flow-col auto-cols-max gap-4'>
+                    <div className='rounded-full bg-setlife w-fit px-2 mx-auto'>
+                        <p className='p-2 text-lg'>
+                            2
+                        </p>
+                    </div>
+                    <p className='m-auto'>
+                        Establish budget
+                    </p>
+                </div>
+                <div className='grid grid-cols-1 gap-4 mt-4'>
+                    <input 
+                        type='text'
+                        placeholder='Budget range'
+                        onChange={(e) => setBudgetRange(parseInt(e.target.value, 10))}
+                        className='
+                        form-control
+                        block
+                        w-full
+                        px-3
+                        py-1.5
+                        text-light
```

Comment:
NIT: Light text is not blending well with the white background
```suggestion
                        text-black
```